File: apzkr-pzpi-21-11-kokhanevych-pavlo/Task1-Server/src/main/java/com/pet/hotel/businessLogic/domain/transferObjects/rent/ScheduleFeedResponse.java
package com.pet.hotel.businessLogic.domain.transferObjects.rent;


import com.fasterxml.jackson.annotation.JsonProperty;
import com.pet.hotel.data.enums.FoodType;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ScheduleFeedResponse {

    private int scheduleId;

    private LocalDateTime dateTime;

    @JsonProperty("portion")
    private float portion;

    private FoodType typeFood;
}

File: apzkr-pzpi-21-11-kokhanevych-pavlo/Task4-MobileClient/src/main/java/com/animal/hotel/domain/models/user/ScheduleFeedResponse.kt
package com.animal.hotel.domain.models.user

import com.animal.hotel.utils.enums.FoodType
import java.time.LocalDateTime

data class ScheduleFeedResponse(
    val scheduleId: Int,
    val dateTime: LocalDateTime,
    val portion: Float,
    val typeFood: FoodType
)
